src/libs/antares/study/parts/hydro/container.cpp

```diff
@@ -248,115 +276,47 @@ bool PartHydro::LoadFromFolder(Study& study, const AnyString& folder)
         return false;
     }
 
-    const char* const sectionName = "inter-daily-breakdown";
-
     IniFile::Section* section;
-    IniFile::Property* property;
 
-    if ((section = ini.find(sectionName)))
+    if ((section = ini.find("inter-daily-breakdown")))
     {
-        if ((property = section->firstProperty))
```

Comment:
(property = section->firstProperty) initialization is missing on new code

Reply:
`loadProperties(study, section->firstProperty,`
given to the function to simplify